### PR TITLE
Multi-input PSBT spend for homogeneous input asset ID

### PR DESCRIPTION
**Pull request by ffranr**
This PR solves https://github.com/lightninglabs/taproot-assets/issues/297 for the case were each input asset ID is the same.

This PR modifies our asset send procedure such that we can spend a multi-input full value PSBT where each input has the same asset ID. The PR adds an itest to test the new changes.

There are more things we should test:
* more than 2 inputs
* sending the received PSBT spend onwards once more.

But lets keep this PR small if we can and open another (if that makes sense).

### CODE REVIEW

File: tapscript/send.go [hunk 1]
```diff
@@ -506,32 +506,51 @@ func PrepareOutputAssets(ctx context.Context, vPkt *tappsbt.VPacket) error {
...
 	if isFullValueInteractiveSend {
-		if len(inputs) != 1 {
-			return fmt.Errorf("full value interactive send " +
-				"must have exactly one input")
+		// Sum the total amount of the input assets.
+		inputsAmountSum := uint64(0)
+		for idx := range inputs {
+			input := inputs[idx]
+
+			// TODO(ffranr): At the moment, we need to ensure that
```
Comment (guggero) [inline tapscript/send.go:514]: Does this need to be a TODO? I think we can just mention here that we do an additional check for clarity.

**Review by guggero — APPROVED** [review]
Very nice, LGTM :tada:

**Comment by ffranr** [comment]
Replaced with https://github.com/lightninglabs/taproot-assets/pull/743

File: tapscript/send.go (hunk 1 above)
Comment (ffranr) [reply tapscript/send.go:514]: Sure, no problem :+1:

**Pull request closed without merging**